assets/src/blocks/Media/MediaFrontend.js

```diff
@@ -0,0 +1,54 @@
+import { MediaEmbedPreview } from './MediaEmbedPreview';
+import { MediaElementVideo } from './MediaElementVideo';
+
+const { __ } = wp.i18n;
+
+const wrapAndRemoveSize = embedHtml => {
+  const wrapperDiv = document.createElement('div');
+  wrapperDiv.innerHTML = embedHtml;
+  if (wrapperDiv.firstChild && wrapperDiv.firstChild.src && wrapperDiv.firstChild.src.indexOf('youtube') > -1) {
+    wrapperDiv.className = 'embed-container';
+    wrapperDiv.firstChild.removeAttribute('width');
+    wrapperDiv.firstChild.removeAttribute('height');
+    return wrapperDiv.outerHTML;
+  } else {
+    return embedHtml;
+  }
+}
+
+export const MediaFrontend = (props) => {
+  const {
+    video_title,
+    description,
+    embedHtml,
+    posterURL,
+    youtube_id,
+  } = props;
+
+  // Deprecate: rename this attribute.
+  const mediaURL = youtube_id;
```

Comment:
Maybe we can already find the new name before we apply it. We use other attributes with their snake case name in the code, so if we make this variable already the new snake case name, we only need to remove this line and extract the value from the props instead.